.guni.env

```diff
@@ -0,0 +1,21 @@
+# DEFAULT_GAS_PRICE=  50 GWEI by default
+
+TOKEN=0x50379f632ca68D36E50cfBC8F78fe16bd1499d1e
+ATOKEN=0x517AD97cD3543eE616cDb3D7765b201D6c9dFFdd
+STABLE_DEBT_TOKEN=0x135bb9dfd7880a53ef86b6f281aa4c3a9addb85c
+VARIABLE_DEBT_TOKEN=0x104e375e7a62ac88317b93a2288865513c1dc511
+INTEREST_STRATEGY=0xfffE32106A68aA3eD39CcCE673B646423EEaB62a
```

Comment:
incorrect?  the one placed at the proposal is: 0x52E39422cd86a12a13773D86af5FdBF5665989aD

Reply:
fixed

Reply:
also note that most of these env variables are not currently being used (they are hardcoded constants in the contract) so they can also be removed if you like.